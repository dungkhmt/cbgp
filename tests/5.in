20 20
20 30
12 5
10 0
7 2
17 10
8 5
15 11
15 0
17 9
19 7
16 5
8 2
7 18
4 9
7 1
9 14
18 0
13 0
6 2
7 3
14 17
18 1
11 10
12 4
5 2
2 0
13 4
14 11
12 13
5 13
18 2
1
0 0 21 1 20 3 30 0 14 6 16 4 21 4 27 0 18 5 13 7 9 7 11 9 15 5 12 4 12 8 10 10 21 9 11 7 20 0 23 4 
