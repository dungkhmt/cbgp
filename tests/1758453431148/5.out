(12, 5), (10, 0), (7, 2), (17, 10), (8, 5), (15, 11), (15, 0), (17, 9), (19, 7), (16, 5), (8, 2), (7, 18), (4, 9), (7, 1), (9, 14), (18, 0), (13, 0), (6, 2), (7, 3), (14, 17), (18, 1), (11, 10), (12, 4), (5, 2), (2, 0), (13, 4), (14, 11), (12, 13), (5, 13), (18, 2), 
numberIntersection=0 
minDistance=5.366563145999495 
minAngle=0.2662520491509257 
sumAngle=17.158321842072567 
sumDistance=17874.36989859732 
minEdgeLength=5.385164807134504 
sumEdgeLength=665.9837733701304 

0.0000000000 5.3665631460 0.2662520492 17.1583218421 17874.3698985973 5.3851648071 665.9837733701
0: (80, 71),
1: (74, 26),
2: (65, 32),
3: (80, 0),
4: (52, 50),
5: (60, 42),
6: (13, 0),
7: (78, 6),
8: (58, 34),
9: (46, 47),
10: (44, 59),
11: (39, 57),
12: (56, 46),
13: (70, 60),
14: (38, 51),
15: (18, 74),
16: (0, 0),
17: (46, 53),
18: (80, 35),
19: (24, 0),
80 71 74 26 65 32 80 0 52 50 60 42 13 0 78 6 58 34 46 47 44 59 39 57 56 46 70 60 38 51 18 74 0 0 46 53 80 35 24 0 
0: (80, 71), 1: (74, 26), 2: (65, 32), 3: (80, 0), 4: (52, 50), 5: (60, 42), 6: (13, 0), 7: (78, 6), 8: (58, 34), 9: (46, 47), 10: (44, 59), 11: (39, 57), 12: (56, 46), 13: (70, 60), 14: (38, 51), 15: (18, 74), 16: (0, 0), 17: (46, 53), 18: (80, 35), 19: (24, 0), 
numberIntersection=0 minAngle=0.2662520491509257 sumAngle=17.15832184208209 minEdgeLength=5.385164807134504 sumEdgeLength=665.9837733701268 minDistance=5.366563145999495 sumDistance=17874.369898598397 
Time taken: 12.987 s
